github/gists.go

```diff
@@ -52,6 +52,30 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
+	Version      *string        `json:"version,omitempty"`
+	User         *User          `json:"user,omitempty"`
+	ChangeStatus map[string]int `json:"change_status,omitempty"`
+	CommitedAt   *Timestamp     `json:"commited_at,omitempty"`
+}
+
+func (gc GistCommit) String() string {
+	return Stringify(gc)
+}
+
+// GistFork represents a fork of a gist
```

Comment:
Nitpick. This sentence does not end with a period. Other comments have sentences that do end with periods, so it'd be nice and more consistent to include a period at the end of this sentence too.